app/components/Search/store/actions/search.actions.js

```diff
@@ -0,0 +1,41 @@
+import Api from '../../../../services/api';
```

Comment:
We are divided in the company on this practice, but in my team I strongly wanted it: try to add a shortcut in your webpack so that you could use absolute paths like "**Presentational**/MyComponent" or "**Components**/TheComponent" - I see many benefits in doing it:

- if you want to move a folder or a file in the files hierarchy of the app you don't need to care about updating paths.
- Imports become really meaningful and readable considering you know what comes from every time.

Reply:
I agree 100% and for a web application I would normally use absolute imports.  But react native doesn't use webpack so aliasing isn't an option and the alternatives that I'm aware of can be problematic.  

- `babel-module-resolver` plugin to set the src paths.  But the issue with that is it will often times cause unexpected behavior with IDE intellisense & code completion without doing major config changes to the IDE.  For the sake of uniformity across dev environments, I'd prefer to avoid that. From my experience, it's also prone to bundling issues due to the caching mechanism under the hood.  That's usually not an issue on web, but due to the way gradle handles caching when i've used it on RN projects in the past it's led to having to constantly rebuild the app, clear cache, etc.  

- dummy `package.json` in each component directory - seems hacky to me and the tradeoff between import legibility vs added architecture complexity isn't justified. 

But if there's another approach to achieve absolute imports, I'm open to ideas on this.